jax_lensing/spectral.py

```diff
@@ -17,15 +17,22 @@ def radial_profile(data):
   radialprofile = tbin / nr
   return radialprofile
 
-def measure_power_spectrum(map_data):
+def measure_power_spectrum(map_data, pixel_size):
   """
   measures power 2d data
-  :param power: map
+  :param power: map (nxn)
+  :param pixel_size: pixel_size (rad/pixel)
+  :return: ell
   :return: power spectrum
+  
   """
   data_ft = jnp.fft.fftshift(jnp.fft.fft2(map_data)) / map_data.shape[0]
   nyquist = np.int(map_data.shape[0]/2)
-  return radial_profile(jnp.real(data_ft*jnp.conj(data_ft)))[:nyquist]
+  power_spectrum_1d =  radial_profile(jnp.real(data_ft*jnp.conj(data_ft)))[:nyquist] * (pixel_size)**2
+  k = np.arange(power_spectrum_1d.shape[0])
```

Comment:
hummmmm maybe we should use fftfreq for this? altough it should return the same result...

Reply:
ok maybe not ^^'